west.yml

```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: a3a6410b55fa97a9c6400c37d362c9137c5050b6
+      revision: pull/389/head
```

Comment:
Update the name-whitelist to bring in trusted-firmware-m.

Reply:
done now